### PR TITLE
Add: Beowulfso.BSU version 0.0.52

### PR DESCRIPTION
**Pull request by Arron17**
- [y ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [y] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [y] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [y] Have you tested your manifest locally with `winget install --manifest <path>`?
- [y] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20322)

### CODE REVIEW

File: manifests/b/Beowulfso/BSU/0.0.52/Beowulfso.BSU.locale.en-GB.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+PackageUrl: https://u.beowulfso.com/
+License: MIT
+LicenseUrl: https://github.com/BeowulfStratOps/BSU.Sync/blob/master/LICENSE.txt
+#Copyright: 
+#CopyrightUrl: 
+ShortDescription: Mod Downloader and Updater for Arma 3
+#Description: 
+Moniker: bsu
+Tags:
+- arma bsu beowulf beowulfso
```
Comment (ItzLevvie) [inline manifests/b/Beowulfso/BSU/0.0.52/Beowulfso.BSU.locale.en-GB.yaml:22]:
```suggestion
- arma
- bsu
- beowulf
- beowulfso
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

File: manifests/b/Beowulfso/BSU/0.0.52/Beowulfso.BSU.locale.en-GB.yaml (hunk 1 above)
Comment (Arron17) [reply manifests/b/Beowulfso/BSU/0.0.52/Beowulfso.BSU.locale.en-GB.yaml:22]: I'll commit the suggestion as it looks neater.

This is how the YamlCreate.PS1 file generated the YAML though with it all on 1 line.

**Review by OfficialEsco — APPROVED** [review]
> This is how the YamlCreate.PS1 file generated the YAML though with it all on 1 line.

I should probably have specified that in the script, its separated by `,` which was default in ManifestVersion 0.1.0

**Pull request merged**